### PR TITLE
Feature/int 1261 tabs components

### PR DESCRIPTION
**Pull request by sabbir0179**

### CODE REVIEW

**Review by Momen-a9s — CHANGES_REQUESTED** [review]
The branch didn't work locally with me, please check the comment in tab stories

File: src/components/navigation/Tabs/TabPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  index: number
+  value: number
+}
+
+export const TabPanel: JSX.Element<TabPanelProps> = (props) => {
+  const { children, value, index, ...other } = props
+  const classes = useStyles()
+
+  return (
+    <div
```
[the remark this answers is not in the document]
Comment (sabbir0179) [reply src/components/navigation/Tabs/TabPanel.tsx:15]: This is a custom TabPanel component designed for the Tabs, as shown in Tabs example. Certain props that were used here weren't supported in Mui_tab_panel component given by default.

File: src/components/navigation/Tabs/Tabs.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,237 @@
+import React from 'react'
+import { withDesign } from 'storybook-addon-designs'
+import { makeStyles, Theme } from '@material-ui/core/styles'
+import { Typography, AppBar, TabsProps, Paper } from '@material-ui/core'
+
+import { Tabs } from './Tabs'
+import { Tab } from './Tab'
+import { TabPanel } from './TabPanel'
+
```
[the remark this answers is not in the document]
Comment (sabbir0179) [reply src/components/navigation/Tabs/Tabs.stories.tsx:9]: fixed

File: src/components/navigation/Tabs/Tabs.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+    }
+  }
+})
+
+export const Default: Story<TabsProps> = (args): JSX.Element => {
+  const classes = useStyles()
+  const [value, setValue] = React.useState<number>(0)
+
+  const handleChange = (
+    event: React.ChangeEvent<{}>,
```
[the remark this answers is not in the document]
Comment (sabbir0179) [reply src/components/navigation/Tabs/Tabs.stories.tsx:64]: fixed it

**Review by Momen-a9s — APPROVED** [review]
nice work, thanks for changes

**Pull request merged**